src/openrct2/rct12/RCT12.h

```diff
@@ -194,6 +214,12 @@ struct RCT12xy8
 };
 assert_struct_size(RCT12xy8, 2);
 
+enum TD46MazeElementType : uint8_t
```

Comment:
Let's make it an enum class, since it is a new enum.

Reply:
Done.